### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GustavoMA**
#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Create a API Docs.
```
Comment (luciannojunior) [inline CHANGELOG.md]: an*

File: README.md [hunk 2]
```diff
@@ -1,7 +1,74 @@
...
+
+To use this app, you need to import in your dependencies on `manifest.json`.
+
+```json
+  "dependencies": {
+    "vtex.login": "2.x"
+  }
+```
+
+Then, add `login` block into our app theme, as we do in our [Store Header app](https://github.com/vtex-apps/store-header/blob/master/store/blocks.json).
```
Comment (vitoriaHeliane) [inline README.md]: `login` or ``login-content

File: README.md [hunk 3]
```diff
@@ -1,7 +1,74 @@
...
+
+### Blocks API
+This app has an interface that describes which rules must be implemented by a block when you want to use the minicart.
```
Comment (GustavoMA) [inline README.md]: `when you want to use the login app.`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,89 @@
...
+```json
+  "dependencies": {
+    "vtex.login": "2.x"
+  }
+```
+
+Then, add `login` or `login-content` block into our app theme, as we do in our [Store Header app](https://github.com/vtex-apps/store-header/blob/master/store/blocks.json).
+
+### Blocks API
+This app has an interface that describes which rules must be implemented by a block when you want to use `Login` or `LoginContent`.
```
Comment (brunojdo) [inline README.md]: Explain the difference between login and login-content blocks

Comment (vitoriaHeliane) [reply README.md]: The same

File: README.md [hunk 5]
```diff
@@ -1,7 +1,130 @@
...
+2. Create a file called `vtex.login.css` inside the `styles/css` folder. Add your custom styles:
+
+```css
+.container {
+  margin-top: 10px;
+}
+```
+
+#### CSS namespaces
+:construction: :construction: :construction:
```
Comment (theomoura) [inline README.md]: use css from #109

Comment (paul0vinicius) [inline README.md]: Use the CSS namespaces from https://github.com/vtex-apps/login/pull/109

File: README.md [hunk 6]
```diff
@@ -1,7 +1,130 @@
 # VTEX Login app 
 
+## Description
+
+The VTEX Login app responsible to handle user login.
```
Comment (vitoriaHeliane) [inline README.md]: Change the description to follow the template.

File: README.md [hunk 7]
```diff
@@ -1,7 +1,168 @@
...
+formLinkContainer | Container for links | [EmailAndPassword](/react/components/EmailAndPassword.js)
+arrowUp | Arrow up | [LoginComponent](/react/components/LoginComponent.js)
+buttonLink | Button link | [LoginComponent](/react/components/LoginComponent.js)
+formError | Error form | [LoginComponent](/react/components/FormError.js)
+content | Login Content container | [LoginContent](/react/LoginContent.js)
+button | Any button | [AccountOptions](/react/components/AccountOptions.js), [LoginOptions](/react/components/LoginOptions.js), [OAuth](/react/components/OAuth.js)
+sendButton | Buttons sending informations | [CodeConfirmation](/react/components/CodeConfirmation.js), [EmailAndPassword](/react/components/EmailAndPassword.js), [EmailVerification](/react/components/EmailVerification.js), [RecoveryPassword](/react/components/RecoveryPassword.js)
+buttonSocial | Button to social media | [OAuth](/react/components/OAuth.js)
+buttonDanger | Danger button | [LoginOptions](/react/components/LoginOptions.js
```
Comment (GustavoMA) [inline README.md]:
```suggestion
buttonDanger | Danger button | [LoginOptions](/react/components/LoginOptions.js)
```

File: README.md [hunk 8]
```diff
@@ -1,7 +1,168 @@
 # VTEX Login app 
 
+## Description
+
+The VTEX Login app responsible to handle user login, and this app is used by store theme.
```
Comment (vitoriaHeliane) [inline README.md]: The VTEX Login app is a store component that is responsible to handle user login, and this app is used by store theme.

Comment (GustavoMA) [reply README.md]: Login isn't a store component

**Pull request merged**